src/index.js

```diff
@@ -35,6 +36,10 @@ export default class SaucelabsConnector {
             retries:    WEB_DRIVER_CONFIGURATION_RETRIES,
             timeout:    WEB_DRIVER_CONFIGURATION_TIMEOUT
         });
+
+        this.options = {
+            showBrowserStartMessage: options.showBrowserStartMessage

```

Comment:
What's about naming it `disableConnectorLogs` or `disableConnectorLogging` to avoid confusion with disabling SauceLabs logs?